README

```diff
@@ -1,3 +1,20 @@
+  _   _      _ _                                                                 
+ | | | | ___| | | ___                                                            
+ | |_| |/ _ \ | |/ _ \                                                           
+ |  _  |  __/ | | (_) |                                                          
+ |_| |_|\___|_|_|\___( )                         ____  _             _ _       _ 
+ | |_   _| |_   _| | |/_ _ __ ___   ___  _ __   / ___|| |_ _   _  __| (_) ___ | |
+ | | | | | | | | | |/ _ \ '_ ` _ \ / _ \| '_ \  \___ \| __| | | |/ _` | |/ _ \| |
+ | | |_| | | |_| | |  __/ | | | | | (_) | | | |  ___) | |_| |_| | (_| | | (_) |_|
+ |_|\__,_|_|\__,_|_|\___|_| |_| |_|\___/|_| |_| |____/ \__|\__,_|\__,_|_|\___/(_)
+                                                                                                                    
+
+For more information about this fork, please see:
+https://github.com/mirrorfitness/mirrorlink-ios/blob/main/README_SSL.md
```

Comment:
This link is dead for now, until we finish merging this into MirrorLink. For now:

https://github.com/mirrorfitness/mirrorlink-ios/blob/NSF-501/README_SSL.md